common/services/prismic/link-resolver.ts

```diff
@@ -52,6 +54,14 @@ function linkResolver(doc: Props | DataProps): string {
   if (type === 'pages') {
     if ('siteSection' in doc) {
       return `${doc.siteSection}/${uid}`;
+    } else if ('tags' in doc) {
+      // Needed for Prismic previews
+      const docSiteSection = doc.tags.find(t => isSiteSection(t));
+
+      const isLandingPage = docSiteSection === uid;
```

Comment:
Do we need to do this for the 'siteSection' in doc case too?

Reply:
We will look into this as a separate PR

Reply:
To be addressed here https://github.com/wellcomecollection/wellcomecollection.org/pull/11354#issuecomment-2447325846

Reply:
[Addressed ](https://github.com/wellcomecollection/wellcomecollection.org/pull/11354/commits/bbe2eec4273f6b8c286488aa44063f8d9b22273c)